test-poc/framework/src/main/java/org/keycloak/test/framework/server/KeycloakTestServerSupplier.java

```diff
@@ -23,8 +23,8 @@ public Class<KeycloakIntegrationTest> getAnnotationClass() {
     public InstanceWrapper<KeycloakTestServer, KeycloakIntegrationTest> getValue(Registry registry, KeycloakIntegrationTest annotation) {
         KeycloakTestServerConfig serverConfig = SupplierHelpers.getInstance(annotation.config());
 
-//        RemoteKeycloakTestServer keycloakTestServer = new RemoteKeycloakTestServer();
-        EmbeddedKeycloakTestServer keycloakTestServer = new EmbeddedKeycloakTestServer();
+        RemoteKeycloakTestServer keycloakTestServer = new RemoteKeycloakTestServer();
+//        EmbeddedKeycloakTestServer keycloakTestServer = new EmbeddedKeycloakTestServer();
```

Comment:
Some test changes that need to be reverted?

Reply:
lol, can't wait to having the ability to select providers :)